runtests.sh

```diff
@@ -4,4 +4,4 @@ echo "Running flake8"
 flake8 || { echo "flake8 errors found!"; exit 1; }
 
 echo "Running Python tests"
-py.test tests/$* --cov-report html --cov treeherder
+DJANGO_SETTINGS_MODULE=tests.settings py.test tests/$* --cov-report html --cov treeherder
```

Comment:
IMO we should set this during test setup, otherwise it's going to mean every time we run `py.test` by hand we have to specify it.